website/integrations/services/openproject/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenProject
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is OpenProject
+
+> OpenProject is a open source project management software.
+>
+> -- https://www.openproject.org/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `openproject.company` is the FQDN of the Service install. (Remove this for SaaS)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+Create a [OAuth2/OpenID provider](https://goauthentik.io/docs/providers/oauth2) with the following parameters:
```

Comment:
```suggestion
Create a OAuth2/OpenID provider with the following parameters:
```
We usually don't like like this, but I'm sure @tanberry will be able to provide some more useful feedback

Reply:
True, we don't usually link to the docs like this from an integration-specific doc, but in this case I think it is fine. The next one, linking to our Applications docs, does not work as well...
So let's leave this link in, but next one, please.